schema/schema.cc

```diff
@@ -821,6 +822,48 @@ sstring schema::element_type(replica::database& db) const {
     return "table";
 }
 
+static void describe_index_columns(std::ostream& os, bool is_local, const schema& index_schema, schema_ptr base_schema) {
```

Comment:
There's something I don't understand here, but I have to be honest, I don't remember the details of what we can or can't be done.

The target **strings** are meant to be used in our on-disk format. But in memory, we have schema and index_manager. Those should give you objects with _index.target_type() and target_column(), so you shouldn't need to process target strings at all... Are you sure you needed to do this?

Reply:
Yes, I saw those functions but they are methods of `cql3::statements::index_target` class. And this class is only used when creating the index and AFAIK it's not used or stored anywhere else.

And to recreate `index_target` objects myself, I would need to parse target string to extract target type and column definition, so there is no point to create the object only to get the arguments I've just used to create it.